client/src/components/dashboard/TemplateSelector.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * This file exports a pop up window, that prompts user
+ * to select a theme in Researchify dashboard page.
+ */
+import React, { useState } from 'react';
+import { Container, Button, Modal, Form, Col, Image } from 'react-bootstrap';
+// import { useDispatch } from 'react-redux';
+// import { updateTeamTheme } from '../../actions/team';
+
+// Picutre of each layout
+import singleColumnLayout from '../../images/single-column-layout.png';
+import fShapeLayout from '../../images/f-shape-layout.png';
+import zigZagLayout from '../../images/zig-zag-layout.png';
+
+
+/**
+ * Form for user input github credentials and select template.
+ */
+const TemplateSelector = (props) => {
+  // const dispatch = useDispatch();
+
+  // Storing and passing Form Inputs, theme1 & layout1 as defualt
+  const [formInputs, setInputs] = useState({
+    layout: 1,
+    primaryColor: '#419aee',
+    secondaryColor: '#8da4d1',
+  });
+
+  const updateForm = (form) => {
+    const { name, value } = form.target;
+    if (name === 'theme') {
+      let primaryColor;
+      let secondaryColor;
+      switch (value) {
+        case 'theme1':
+          primaryColor = '#419aee';
```

Comment:
If these themes are going to be stored in the db, then you shouldn't need to pass the colors into setInputs(), just the theme id (ie theme1/theme2/theme3)?

Reply:
These colour combinations are not finalised yet and in our `theme.model` database, `themeId` will represent the combination of layout and colours. We don't have a individual model to store colour combinations. If we're to give each colour combination an id, I reckon we might need to manually put them into the database? The reason I put it here is mainly because we can change them from the front end and create a new theme in database through API calls.

Reply:
Yeah, otherwise you would have to insert them manually into the database anyway. This is fine for now then!